pkg/builder/container_builder.go

```diff
@@ -105,3 +105,13 @@ func (b *ContainerBuilder) PullPolicy(pullPolicy corev1api.PullPolicy) *Containe
 	b.object.ImagePullPolicy = pullPolicy
 	return b
 }
+
+func (b *ContainerBuilder) Command(command []string) *ContainerBuilder {
+	if b.object.Command == nil {
+		b.object.Command = []string{}
+	}
+	for _, c := range command {
```

Comment:
Could the loop be compressed into

```go
b.object.Command = append(b.object.Command, c...)
```

?

Reply:
fixed